src/app/home-page/widget/entry-box/entry-box.component.ts

```diff
@@ -77,29 +79,27 @@ export class EntryBoxComponent extends Base implements OnInit {
   }
 
   getMyEntries() {
-    this.usersService
-      .getUserEntries(null, this.filterText)
+    if (this.entryType === EntryUpdateTime.EntryTypeEnum.TOOL) {
```

Comment:
A suggestion -- calculate the type, then have the `getUserEntries` call written only once. It makes it easier, for example, if we add an additional parameter some day to `getUserEntries`, we'd only need to update the code in one place in this file.

Example, untested:

```
const entryTypeParam = this.entryType + "S"; // This is a little hacky
this.usersService.getUserEntries(null, this.filterText, entryTypeParam)
  .pipe(...
```

The code is also a little easier to read IMHO, you don't need the `userEntries$` property...